custom/covid/management/commands/update_cases.py

```diff
@@ -27,11 +28,25 @@ def find_case_ids_by_type(self, domain, case_type):
         print(f"Found {len(case_ids)} {case_type} cases in {domain}")
         return case_ids
 
+    def log_data(self, domain, command, case_type, total_cases, num_updated, failed_cases, loc_id=None):
```

Comment:
I think you can use `__name__` in this function rather than passing in `command`.

Reply:
I tested this and it printed the `update_cases` :/. Decided to leave as is for now

Reply:
Fair.